app/models/downloaded_file.rb

```
@@ -1,12 +1,14 @@
 class DownloadedFile < ApplicationRecord
 
-  def self.add_file!(checksum_type, checksum, file_size, local_path)
-    return nil unless local_path.match?(/\.(rpm|drpm)$/)
+  def self.add_file(checksum_type, checksum, file_size, local_path)
+    return unless local_path.match?(/\.(rpm|drpm)$/)
 
     DownloadedFile.create({ checksum_type: checksum_type,
```

Can we use find_or_create here to avoid the empty catch block?